demos/python_demos/seq2seq_translation/README.md

```diff
@@ -24,6 +26,53 @@ cd demos/python_demos/seq2seq_translation
 docker run -it --rm -p 9000:9000 -v ${PWD}/servable:/workspace openvino/model_server:py --config_path /workspace/config.json --port 9000
 ```
 
+## Running Model Server on Windows (experimental)
+
+### Build Model Server binary with Python enabled
+For the Model Server to include embedded Python interpreter you need to [build the binary from scratch](../../../docs/windows_developer_guide.md). Note the `--define MEDIAPIPE_DISABLE=0 --define PYTHON_DISABLE=0` part in `bazel build` command that specifies that both MediaPipe and Python support will be included (Python support implementation is based on MediaPipe). 
+
+Next steps assume you have followed the building instruction from the developer guide. If you chose different download or installation paths you will need to adjust below instructions.
+
+### Prepare environment for Python servable deployment
+
+Once you have OVMS executable, there are three more steps for the Python servable to work.
+
+1. Create a copy of `pyovms.so` file created during the build with `.pyd` extension in the same location. 
```

Comment:
I think we need to automate this step in our windows build.

Reply:
Let's do this in a separate PR, so we don't mix too many topics in this one